task_manager/api.py

```diff
@@ -7,130 +7,130 @@
 import task_manager.routes.users as users
 import task_manager.routes.auth as auth
 
-
 user_ns = Namespace('user', description='User API endpoints')
 task_ns = Namespace('task', description='Task API endpoints')
 
 
 @apis.route('/test', methods=['GET'])
 class Test(Resource):
-	def get(self):
-		return "Hello World"
+    def get(self):
+        return "Hello World"
 
 
 @apis.route('/')
 @apis.route('/dashboard')  # dashboard
 class Index(Resource):
-	'''If logged in display index, else display homepage'''
+    '''If logged in display index, else display homepage'''
```

Comment:
is this a spaces vs tabs thing?

Reply:
flake8 was throwing lint errors since our file had mix of spaces and tabs. i ran a global cleanup to convert all tabs to spaces according to flake8 convention.

Reply:
in the future I would prefer all the spaces be turned into tabs, our flake8 ignores space errors because tabs were preferred